plugins/modules/junos_bgp_global.py

```diff
@@ -1442,6 +1477,13 @@
 
 # After state
 # ----------
+# admin# show protocols bgp
+#
+# [edit]
+# admin# show routing-options autonomous-system
+#
+#[edit]
+
 
```

Comment:
In the example with parsed, the xml has as_number set https://github.com/ansible-collections/junipernetworks.junos/blob/c47e270ebd9d0e7eafc6c4cac0bad804c32305af/plugins/modules/junos_bgp_global.py#L1643, but the same is not shown in parsed data.

Reply:
yeah, I'll update the parsed opeartion